package.json

```diff
@@ -24,21 +24,27 @@
     "lint": "eslint .",
     "pretest": "npm run lint",
     "test": "mocha --async-only",
-    "cover": "nyc --reporter=lcov --reporter=text-summary npm test",
-    "coveralls": "npm run cover && istanbul-coveralls",
+    "cover": "nyc mocha --async-only",
```

Comment:
I think this is wrong. We move the nyc into the "test" script, right?

Reply:
Got it. I'll modify it.